.vscode/launch.json

```diff
@@ -84,6 +84,42 @@
                 "-gt"
             ]
         },
+        {
```

Comment:
IMHO no IDE specific files should be part of the repo. One usually does not spot any such files in other repositories. It's constantly clashing with my file when I am adding some quick launch snippets. I can deal with this, though ;)

Reply:
what line do you mean in particular?
I'm with you that there should not (or only few) be OS specific files/settings in the repo.
For me, the launch config work in both macOS and Windows without a problem using VSCode.

Reply:
read it a second time and you mean the whole file because it is IDE - VSCode - specific?
For me it is convenient if I have the same launch configs on both of my computers and additionally there is a launch config which is used as part of unittests.
Is is clashing because you change the launch.json and Git wants you to commit the changes?

Reply:
Yap, I meant the whole file. I know it is convenient, but only for one person working on the project. But no biggy. I'll just refrain from adding some on my own.

Reply:
I got your point and this is valid in my opinion when many users develop a project. Since we are only a few active people and know how to get around it I would leave the file in.
In my opinion feel free to add entries here so we all have them.